operations/CSM_product_management/Configure_Non_Compute_Nodes.md

```diff
@@ -0,0 +1,168 @@
+
+Local site security requirements may preclude use of passwordless SSH access. If
+this is the case, remove or comment out the invocation of the
+`trust-csm-public-keys` role in Ansible plays in the configuration repositories
+of the environments where it is configured. By default, the HPE Cray Operating
+System (COS) and User Access Node (UAN) configurations enable passwordless SSH.
+Consult the product documentation for each environment to change the
+default configuration. Modifying Ansible plays in a configuration repository
+will require a new commit and subsequent update of the
+[configuration layer](../configuration_management/Configuration_Layers.md)
+associated with the product.
+
+> __NOTE__: CFS itself does not use the CSM-provided (or user-supplied) SSH keys
+> to make connections between nodes. CFS will continue to function if
+> passwordless SSH is disabled between CSM and other product environments.
+
+### Restore CSM-provided SSH Keys
+
+> Use this procedure if switching from custom keys to the default CSM SSH keys
+> only, otherwise it can be skipped.
+
+The `csm-ssh-keys` Kubernetes deployment provided by CSM periodically checks the
+configmap and secret containing the key information. If these entries do not
+exist, it will recreate them from the default CSM keys. In this case, deleting
+the associated configmap and secrets will republish them with the default
+CSM-provided keys.
+
+1. Delete the `csm-private-key` Kubernetes secret.
+   ```bash
+   ncn# kubectl delete secret -n services csm-private-key
+   ```
+1. Delete the `csm-public-key` Kubernetes configmap.
+   ```bash
+   ncn# kubectl delete configmap -n services csm-public-key
+   ```
+
+<a name="set_root_password"></a>
+## Set the Root Password
+
+The root password is managed on NCNs by using the `csm.password` Ansible role
+located in the CSM configuration management repository. Root passwords are set
+and managed in Vault.
+
+To set the password in Vault, follow steps 1-3 in the
+[Update NCN Passwords](../security_and_authentication/Update_NCN_Passwords.md)
+procedure and then run [NCN personalization](#run_ncn_personalization).
+
+<a name="run_ncn_personalization"></a>
+## Run NCN Personalization
+
+After completing the previous procedures, apply the configuration to the NCNs
+by running NCN personalization with [CFS](../configuration_management/Configuration_Management.md).
+
+Prior to running NCN personalization, gather the following information: 
+
+* HTTP clone URL for the configuration repository in [VCS](../configuration_management/Version_Control_Service_VCS.md),
+* Path to the Ansible play to run in the repository,
+* Commit ID in the repository for CFS to pull and run on the nodes.
+
+1. Get the location of the configuration repository. For CSM, this is
+
+       https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+
+1. Get the Ansible play to run (`site.yml` is the default play to run for CSM):
+
+       site.yml
```

Comment:
FYI Don Bahls had the suggestion that each product list the above sort of info in the same manner to be consistent across products.  See the tables at https://stash.us.cray.com/projects/SHASTA-OS/repos/cos-docs/browse/portal/developer-portal/install/COS_CFS_product_layers_for_NCN_and_CN.md for the format COS went with in case you want to follow.  I have not reviewed the text in that doc closely, but the tables look OK to me FWIW.

Reply:
Replaced list with a table.